adhocteam/Head-Start-TTADP
# [TTAHUB-343] Remove feature flag preventing all from accessing legacy reports in CSV

## Description of change
- Remove feature flag preventing everyone from downloading legacy CSV reports.
- Add a label at the top of the CSV that says "Reports done before March 1, 2021 may have blank fields. These were done in a SmartSheet, not the TTA Hub."

## How to test

- Verify that CSV downloads work
- Verify that they can include legacy reports
- Verify that the warning appears and doesn't interfere with CSV formatting

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-343


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
